src/styles/style.sass

```diff
@@ -62,9 +62,8 @@ body
   display: grid
   grid-template-rows: auto 1fr auto
 
-footer
```

Comment:
Are you removing this because it has no effect?
Seems a bit weird, I think this again play into the `display:grid` on the `body`.

Reply:
Before I changed this, the footer was displaying over the hero image when I removed the wrapper `<div>` around the `<main>`, which I did to allow the whole page to share the responsiveness